apps/studio/components/interfaces/Database/Wrappers/EditWrapper.tsx

```diff
@@ -338,7 +338,7 @@ const EditWrapper = () => {
                     ) : (
                       <div className="space-y-2">
                         {wrapperTables.map((table, i) => {
-                          const label = wrapperMeta.tables[table.index].label
+                          const label = wrapperMeta.tables[table.index]?.label
```

Comment:
If the table is not listed in wrapperMeta, you get a weird colon:
<img width="487" alt="image" src="https://github.com/supabase/supabase/assets/26616127/04ee6836-f50b-45be-9ef1-ce17607a80a1">

I guess a more general question, is there even a reason to go back and look up the constants to find the label? There are a lot of allowed tables in the Stripe FDW that don't seem to be included in the constants, so this will often be missing, and for the ones that are included, the label doesn't seem to add any information that's not immediately obvious from the table name (which is in the line immediately above). Would it be simpler to just drop it? Or rename it `Columns:`, which might be more descriptive?

Reply:
^ good point here, probably don't need to do the round trip for label - looking into it